### PR TITLE
Antag bases minor fixes

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

changes:
  - bugfix: "Fixes burglar shuttle airlock being closed at round start."
  - bugfix: "Fixes some broken icons on the antag bases."
  - bugfix: "Fixes some old style or destructible windows on the antag bases."

!fixes 18259
fixes https://github.com/Aurorastation/Aurora.3/issues/18259

### CODE REVIEW

**Comment by DreamySkrell** [comment]
fixes #18259

**Comment by DreamySkrell** [comment]
!bugfix

**Comment by DreamySkrell** [comment]
!review

File: maps/sccv_horizon/sccv_horizon-4_centcomm.dmm [hunk 1]
```diff
@@ -20952,17 +20952,15 @@
...
-	name = "plating"
+/turf/unsimulated/wall/fakeairlock{
+	icon = 'icons/obj/doors/basic/single/external/door.dmi';
+	icon_state = "door_locked";
+	name = "external airlock";
+	opacity = 0;
+	permit_ao = 0;
+	dir = 2;
+	pixel_x = -16;
+	pixel_y = -16
```
Comment (FluffyGhoster) [inline maps/sccv_horizon/sccv_horizon-4_centcomm.dmm:20963]: This looks absolutely atrocious, there has to be a better and sensibly doable way to do this

File: maps/sccv_horizon/sccv_horizon-4_centcomm.dmm [hunk 2]
```diff
@@ -38499,21 +38357,17 @@
...
-	maxhealth = 140
+/turf/unsimulated/wall/fakeairlock{
+	icon = 'icons/obj/doors/basic/single/external/door.dmi';
+	icon_state = "door_locked";
+	name = "external airlock";
+	opacity = 0;
+	permit_ao = 0;
+	dir = 2;
+	pixel_x = -16;
+	pixel_y = -16
```
Comment (FluffyGhoster) [inline maps/sccv_horizon/sccv_horizon-4_centcomm.dmm:38368]: Same

File: maps/sccv_horizon/sccv_horizon-4_centcomm.dmm (hunk 1 above)
Comment (DreamySkrell) [reply maps/sccv_horizon/sccv_horizon-4_centcomm.dmm:20963]: as below

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**